Dockerfile-dev

```diff
@@ -0,0 +1,43 @@
+FROM java:openjdk-8
+
+#Install sbt
+ENV SBT_VERSION 0.13.11
+RUN \
+  curl -L -o sbt-$SBT_VERSION.deb http://dl.bintray.com/sbt/debian/sbt-$SBT_VERSION.deb && \
+  dpkg -i sbt-$SBT_VERSION.deb && \
+  rm sbt-$SBT_VERSION.deb && \
+  apt-get update && \
+  apt-get install sbt && \
+  rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+# Install Node.js
+RUN \
+    curl -sL https://deb.nodesource.com/setup_6.x | bash - && \
+    apt-get update && \
+    apt-get install -y --no-install-recommends build-essential nodejs && \
+    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+# Install grunt
+RUN npm i -g grunt-cli
+
+# Create config files
+RUN mkdir -p /etc/gu/ && echo 'STAGE=DEV' > /etc/gu/install_vars
+
+VOLUME /frontend
+
+WORKDIR /frontend
+
+# Build app once to populate node and sbt cache
+ADD . /source
+RUN cd /source; make install compile
+RUN  cd /source; ./sbt compile 
+RUN rm -rf /source
```

Comment:
Make these a single `RUN` command, so that docker can cache it as a single layer.